orc8r/lib/go/service/logcodec.go

```diff
@@ -24,15 +27,62 @@ import (
 	grpc_proto "google.golang.org/grpc/encoding/proto"
 )
 
+// logModes
+type grpcLogVerbosityLevel int
+
+const (
+	GRPCLOG_DISABLED    = grpcLogVerbosityLevel(iota) // Default value
+	GRPCLOG_FULL                                      // Prints all protps
+	GRPCLOG_HIDEVERBOSE                               // Prints all except verboseProtos
+	grpclog_end
+)
+
+var (
+	// Default list of verbose protos
+	verboseProtos = map[string]grpcLogVerbosityLevel{
+		"protos.Void":                  GRPCLOG_HIDEVERBOSE,
+		"protos.ServiceInfo":           GRPCLOG_HIDEVERBOSE,
+		"protos.HealthStatus":          GRPCLOG_HIDEVERBOSE,
+		"protos.MetricsContainer":      GRPCLOG_HIDEVERBOSE,
+		"protos.SubmitMetricsRequest":  GRPCLOG_HIDEVERBOSE,
+		"protos.SubmitMetricsResponse": GRPCLOG_HIDEVERBOSE,
+		"grpc.MetricFamilies":          GRPCLOG_HIDEVERBOSE,
+		"grpc.Void":                    GRPCLOG_HIDEVERBOSE,
+	}
```

Comment:
Not sure if I have a recommendation here, but I do want to point out that with the way things are currently structured, we might need to include `/lte` protobufs, or `/feg` protobufs, and those dependencies should typically go the other way, at least based on how `xjtian` originally intended the plugin system to be used.

cc: @hcgatewood for visibility

Reply:
Since this is a debug tool, I wanted to pollute as less as possible service with extra modules, that is why I used hardcoded strings instead of structs itself.

But you guys are ok with those dependencies under lib  I can add those there.